src/Game.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Random;
+
+public class Game {
+
+    private Rules rules;
+    private SYMBOL aiPlayer;
+    private SYMBOL winningSymbol;
+
+    public Game() {
+        aiPlayer = null;
+        rules = new Rules();
+        SYMBOL.setOrdinalToSymbol();
+    }
+
+    public SYMBOL generateAIMove() {
+        int range = SYMBOL.ordinalToSymbol.size();
+        aiPlayer = getRandomSymbolInRange(range);
+        return aiPlayer;
+    }
+
+    SYMBOL play(SYMBOL symbol1, SYMBOL symbol2) {
+        return rules.decideWinner(symbol1, symbol2);
+    }
+
+    public SYMBOL startGame(SYMBOL inputMove) {
+        winningSymbol = play(inputMove, aiPlayer);
+        return winningSymbol;
+    }
+
+    private SYMBOL getRandomSymbolInRange(int range) {
+        int randomOrdinal = randomNumberInRange(randomFractionFromRange(range));
+        return SYMBOL.getSymbolFromOrdinal(randomOrdinal);
+    }
+
+    long randomFractionFromRange(long range) {
+        Random randomGenerator = new Random();
+        return (long) (range * randomGenerator.nextDouble());
+    }
+
+    int randomNumberInRange(long fraction) {
+        int start = 1;
+        return (int) (fraction + start);
+    }
+
+    public SYMBOL getWinner() {
+        return winningSymbol;
+    }
+}
```

Comment:
Keep the level of visibilitiy for methods consistent. `public` to the top, I would then say `private` and then the package private ones. In fact, looking at the tests two of the methods you have as package private (`randomFractionFromRange`, `randomNumberInRange`) are not used at all and could be marked as `private`. `play` is package private but judging by the test, it seems it could/should be public.